rtt/typekit/BoolTypeInfo.hpp

```diff
@@ -49,10 +49,10 @@ namespace RTT
         /**
          * Write boolean as 'true' or 'false'.
          */
-        struct BoolTypeInfo: public TemplateTypeInfo<bool,true>
+        struct BoolTypeInfo: public StdTypeInfo<bool>
         {
             BoolTypeInfo() :
-                TemplateTypeInfo<bool,true> ("bool")
+                StdTypeInfo<bool> ("bool")
             {
```

Comment:
I've tested this, with a component that includes rtt/typekit/Types.hpp and links with the rtt-typekit (the typical usage), and it works == ABI compatible at a user level. I verified with nm and BoolTypeInfo is nowhere referenced in the component library, only in the rtt-typekit library itself.